### DISCUSSION TITLE
Test results upto commit id d7bcf23

### DISCUSSION THREAD

**Discussion by Muralix** [OFS/linux-dfl-backport#59] [General]
@rweight, @rchriste1 : I have validated this branch on Darby creek card with all latest patches picked recently.

I am attaching the Test log for our reference. Please share your feedback if the report looks fine. So then we can merge this into Release branch. 
[ofs_bat_EA_D5005_10_10_22_03_47_non_security_test.log](https://github.com/OPAE/linux-dfl-backport/files/9764922/ofs_bat_EA_D5005_10_10_22_03_47_non_security_test.log)

**Comment by rchriste1** [comment]
I have validated this commit with 2 x N5010 cards in server. Works ok.

**Comment by rweight** [comment]
The changes look good to me @Muralix . I would like to keep the commit IDs the same between the intel-test/fpga-ofs-dev-5.15-lts and intel/fpga-ofs-dev-5.15-lts branches. You should be able to mirror the changes over by doing something like:

git push backport-github d7bcf23c8e9b:intel/fpga-ofs-dev-5.15-lts

... assuming that "backport-github" is the name of your remote. Let me know if you have any questions.
